src/ocaml/preprocess/408/parser_raw.mly

```diff
@@ -314,10 +314,10 @@ let bigarray_set ~loc arr arg newval =
                         Nolabel, newval]))
 
 let lapply ~loc p1 p2 =
-  if !Clflags.applicative_functors
-  then Lapply(p1, p2)
-  else raise (Syntaxerr.Error(
-                  Syntaxerr.Applicative_path (make_loc loc)))
+  if not !Clflags.applicative_functors then
+    raise_error (Syntaxerr.Error(
+      Syntaxerr.Applicative_path (make_loc loc)));
+  Lapply(p1, p2)
```

Comment:
Can you avoid changing the style of the code for patches like this one?
I think keeping the diff against upstream to a minimal is really beneficial.

Reply:
I change the style because the control flow changes. Do you prefer to duplicate the `Lapply (p1, p2)`? I think this is less readable.

Reply:
Ah, I had stupidly assumed that `raise_error` wasn't return, while in fact it is.
Nevermind then.